### ISSUE TITLE
Cannot import sip installed with PyQt5, build fails with import error

### ISSUE THREAD

**Issue by ronaldoussoren** [ronaldoussoren/py2app#274]
**[Original report](https://bitbucket.org/ronaldoussoren/py2app/issue/274) by Sam Schott (Bitbucket: [Sam Schott](https://bitbucket.org/Sam Schott), ).**

----------------------------------------

Trying to build an app with PyQt5 fails because of an `ImportError: No module named sip`. The full traceback is:

```
Traceback (most recent call last):
  File "setup_py2app.py", line 18, in <module>
    setup_requires=['py2app'],
  File "/usr/local/lib/python3.7/site-packages/setuptools/__init__.py", line 145, in setup
    return distutils.core.setup(**attrs)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/usr/local/lib/python3.7/site-packages/py2app/build_app.py", line 838, in run
    self._run()
  File "/usr/local/lib/python3.7/site-packages/py2app/build_app.py", line 1053, in _run
    self.run_normal()
  File "/usr/local/lib/python3.7/site-packages/py2app/build_app.py", line 1150, in run_normal
    self.process_recipes(mf, filters, flatpackages, loader_files)
  File "/usr/local/lib/python3.7/site-packages/py2app/build_app.py", line 1007, in process_recipes
    rval = check(self, mf)
  File "/usr/local/lib/python3.7/site-packages/py2app/recipes/qt5.py", line 22, in check
    mf.import_hook('sip', m)
  File "/usr/local/lib/python3.7/site-packages/modulegraph/modulegraph.py", line 1082, in import_hook
    q, tail = self._find_head_package(parent, name, level)
  File "/usr/local/lib/python3.7/site-packages/modulegraph/modulegraph.py", line 1184, in _find_head_package
    raise ImportError("No module named " + qname)
ImportError: No module named sip
```

I believe this error occurs because PyQt5-sip is installed into the PyQt5 package itself. The above error can be fixed by correctly specifying the module level in the qt5 recipe:

```
mf.import_hook('sip', m, level=1)
```

instead of 

```
mf.import_hook('sip', m)
```

‌

**Comment by ronaldoussoren** [comment]
This should be fixed in py2app 0.20

**Comment by Kentzo** [comment]
@ronaldoussoren Was it fixed? The sip recipe fails to import sip for me (it's available under PyQt5.sip).
